webapp/src/components/Creators.vue

```diff
@@ -3,7 +3,7 @@
     <span v-if="showBubble">
       <UserBubble v-if="showBubble" :creator="creator" :size="size" />
     </span>
-    <span v-if="showNames && creator.display_name" class="display-name">
+    <span v-if="showNames && creator.display_name && creators.length === 1" class="display-name">
       {{ creator.display_name }}
       <span v-if="creator !== creators[creators.length - 1]">,</span>
```

Comment:
It looks like now line 8 will never be reached, since the names will only be shown if only a single creator is there. If multiple creators are provided, and "showNames" is True, no names will be shown, which is confusing (assuming I'm reading this right)

I think it would be better to leave the functionality to show multiple creator names in. Then, cases where we want to show the name only if creators.length===1 can either be handled by adding another prop 'showNameIfOnlyOneCreator`, or just handled whenever creators is called:
```<Creators :creators="ItemCreators" :size="36" showNames="ItemCreators.length === 1"/>```

Reply:
This also allows us to do things like: `<Creators :creators="ItemCreators" :size="36" showNames="ItemCreators.length <= 3"/>